zingolib/src/commands/utils.rs

```diff
@@ -1,16 +1,20 @@
+#![forbid(unsafe_code)]
+//! Currently this mod contains a utility fn that's been factored out of SendCommand
```

Comment:
I think there is a good chance additional functions could get added and get through review without realising this comment is now out of date.  Maybe its best the doc comment talks generally about what this mod is for?

```suggestion
//! Module containing utility functions for the commands interface
```